src/EnergyPlus/ChillerGasAbsorption.cc

```diff
@@ -833,8 +833,11 @@ void GasAbsorberSpecs::setupOutputVariables(EnergyPlusData &state)
                         ChillerName);
 }
 
-void GasAbsorberSpecs::oneTimeInit(EnergyPlusData &state)
+void GasAbsorberSpecs::oneTimeInit_new(EnergyPlusData &state)
 {
+
+    this->setupOutputVariables(state);
```

Comment:
Moved other functions that were under the oneTimeInit boolean flags to the oneTimeInit_new function. Then removed those unused boolean flags.